examples/step-12/step-12.cc

```diff
@@ -120,9 +120,9 @@ namespace Step12
   {
     Assert(dim >= 2, ExcNotImplemented());
 
-    Point<dim> wind_field;
-    wind_field(0) = -p(1);
-    wind_field(1) = p(0);
+    Tensor<1, dim> wind_field;
```

Comment:
Yes, nice. This was definitely a semantic bug before.

Reply:
Same thing is still an issue in `step-12b`

Reply:
Indeed, just pushed the next change... it's a pity that compilation breaks after every such a warning and I don't have an Apple compiler locally to speedup this process a little bit.